docs/pipenv_readme.md

```diff
@@ -0,0 +1,53 @@
+# Presidio analyzer - Pipenv installation
+[Pipenv](https://pipenv.readthedocs.io/en/latest/) is a Python workflow manager, handling dependencies and environment for python packages. This tutorial describes how to install presidio-analyzer locally for development purposes.
+### 1. Install pipenv
+#### Using Pip:
+```
+$ pip install --user pipenv
+```
+#### Homebrew
+```
+$ brew install pipenv
+```
+
+Additional installation instructions: https://pipenv.readthedocs.io/en/latest/install/#installing-pipenv
+
+
+### 2. Create virtualenv for the project
+From the python project's root folder, run:
+```
+$ pipenv shell
+```
+
+### 3. Install all requirements in the Pipfile, including dev requirements
+
+```
+$ pipenv install --dev --sequential
+```
+
+### 4. Run all tests
+```
+$ pipenv run pytest
```

Comment:
After pipenv shell, the environment is activated, and no need to use `pipenv`. You can just call `pytest` inside the virtual environment.

Reply:
Same remark for other pipenv in the rest of the file

Reply:
@omri is it more accurate to switch between 2 and 3. meaning first installing then entering the shell?

Reply:
fixed